modules/i18n/input-method/fcitx.nix

```
@@ -0,0 +1,48 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.i18n.inputMethod.fcitx;
+  fcitxPackage = pkgs.fcitx.override { plugins = cfg.engines; };
+  fcitxEngine = types.package // {
+    name = "fcitx-engine";
+    check = x:
+      (lib.types.package.check x)
```

```suggestion
      (types.package.check x)
```
Small simplification.